<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassDialog</class>
 <widget class="QDialog" name="changePassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游댏 Cambiar Contrase침a</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ============================================
   DI츼LOGO - TEMA NE칍N CYBERPUNK
   ============================================ */

QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #0a0e27, stop:0.5 #16213e, stop:1 #0a0e27);
    border: 3px solid #7b2ff7;
    border-radius: 15px;
}

/* ============================================
   LABEL T칈TULO
   ============================================ */

QLabel#label_userchangepass {
    color: #00ffff;
    font-size: 18px;
    font-weight: bold;
    font-family: 'Segoe UI', Arial, sans-serif;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 rgba(123, 47, 247, 150),
                                stop:0.5 rgba(0, 212, 255, 150),
                                stop:1 rgba(255, 0, 110, 150));
    border: 2px solid #00d4ff;
    border-radius: 10px;
    padding: 10px;
}

/* ============================================
   LINE EDITS (CAMPOS DE TEXTO)
   ============================================ */

QLineEdit {
    background: rgba(26, 35, 62, 180);
    border: 2px solid #7b2ff7;
    border-radius: 12px;
    padding: 10px 15px;
    color: #ffffff;
    font-size: 13px;
    font-family: 'Segoe UI', Arial, sans-serif;
    selection-background-color: #00d4ff;
    selection-color: #000000;
}

QLineEdit:focus {
    border: 2px solid #00d4ff;
    background: rgba(26, 35, 62, 220);
    box-shadow: 0 0 20px rgba(0, 212, 255, 0.6);
}

QLineEdit:hover {
    border: 2px solid #9351ff;
    background: rgba(26, 35, 62, 200);
}

QLineEdit::placeholder {
    color: rgba(255, 255, 255, 100);
    font-style: italic;
}

/* ============================================
   DIALOG BUTTON BOX
   ============================================ */

QDialogButtonBox {
    background: transparent;
}

/* Bot칩n OK/Aceptar */
QDialogButtonBox QPushButton[text="OK"],
QDialogButtonBox QPushButton[text="Aceptar"] {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #00d4ff, stop:1 #7b2ff7);
    border: 2px solid #00d4ff;
    border-radius: 12px;
    padding: 10px 25px;
    color: white;
    font-size: 13px;
    font-weight: bold;
    font-family: 'Segoe UI', Arial, sans-serif;
    min-width: 100px;
    min-height: 35px;
}

QDialogButtonBox QPushButton[text="OK"]:hover,
QDialogButtonBox QPushButton[text="Aceptar"]:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #00e5ff, stop:1 #9351ff);
    box-shadow: 0 0 25px rgba(0, 212, 255, 0.8);
}

QDialogButtonBox QPushButton[text="OK"]:pressed,
QDialogButtonBox QPushButton[text="Aceptar"]:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #00a8cc, stop:1 #6020dd);
    padding-top: 12px;
    padding-bottom: 8px;
}

/* Bot칩n Cancel/Cancelar */
QDialogButtonBox QPushButton[text="Cancel"],
QDialogButtonBox QPushButton[text="Cancelar"] {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #ff006e, stop:1 #ff4444);
    border: 2px solid #ff006e;
    border-radius: 12px;
    padding: 10px 25px;
    color: white;
    font-size: 13px;
    font-weight: bold;
    font-family: 'Segoe UI', Arial, sans-serif;
    min-width: 100px;
    min-height: 35px;
}

QDialogButtonBox QPushButton[text="Cancel"]:hover,
QDialogButtonBox QPushButton[text="Cancelar"]:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #ff1a7a, stop:1 #ff6666);
    box-shadow: 0 0 25px rgba(255, 0, 110, 0.8);
}

QDialogButtonBox QPushButton[text="Cancel"]:pressed,
QDialogButtonBox QPushButton[text="Cancelar"]:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #cc0058, stop:1 #cc3333);
    padding-top: 12px;
    padding-bottom: 8px;
}

/* ============================================
   EFECTOS ADICIONALES
   ============================================ */

QLineEdit#lineEdit_username {
    border-left: 4px solid #00d4ff;
}

QLineEdit#lineEdit_password {
    border-left: 4px solid #ff006e;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>330</width>
     <height>50</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>370</width>
     <height>45</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>游 Repita nueva contrase침a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>370</width>
     <height>45</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>游댐 Nueva contrase침a</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_userchangepass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>370</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>游댏 Cambiar Contrase침a de Usuario</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>changePassDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>changePassDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>